packages/jaspr/lib/src/server/options.dart

```diff
@@ -18,8 +18,8 @@ class Jaspr {
 
   static bool get isInitialized => _options != null;
 
-  static JasprOptions get options => _options ?? JasprOptions();
-  static JasprOptions? _options;
+  static ServerOptions get options => _options ?? ServerOptions();
```

Comment:
Since I saw you made the class constant below:

```suggestion
  static ServerOptions get options => _options ?? const ServerOptions();
```